ui/src/jobs/JobList.tsx

```diff
@@ -24,6 +24,10 @@ const JobFilter = (props: any) => (
             { id: 'success', name: 'Success' },
             { id: 'failed', name: 'Failed' },
         ]} />
+        <SelectInput source="disabled" choices={[
```

Comment:
Why not use `BooleanInput`? the we could remove this code too https://github.com/distribworks/dkron/pull/923/files#diff-5172cc423c1e538ec337bcdf8abf914469f842f13e0c654a55aed5a58ab93460R175

Reply:
True, it is indeed way cleaner